### DISCUSSION TITLE
openrouter support

### DISCUSSION THREAD

**Discussion by grimaldiss** [ScrapeGraphAI/Scrapegraph-ai#274] [Ideas]
the support for openrouter.ai.

**Comment by itsperini** [comment]
Hey @grimaldiss how would you like it to be implemented? Right now I saw it is possible to use the OpenAI class, you can try with that in the graph configuration
![image](https://github.com/VinciGit00/Scrapegraph-ai/assets/19621159/3d6b9f56-d9eb-43a0-ac4d-5f375ff9df40)

Check this example to get an idea on how to modify the config [here](https://github.com/VinciGit00/Scrapegraph-ai/blob/main/examples/deepseek/smart_scarper_deepseek.py)

Let me know!

**Comment by grimaldiss** [comment]
Thank you it works!
